projects/inception-lrt/index.js

```diff
@@ -1,14 +1,30 @@
-async function tvl(api) {
+const ADDRESSES = require('../helper/coreAssets.json')
+
+async function tvl_eth(api) {
   const vaults = ['0x36B429439AB227fAB170A4dFb3321741c8815e55', '0xfE715358368416E01d3A961D3a037b7359735d5e', '0x90E80E25ABDB6205B08DeBa29a87f7eb039023C2', '0x4878F636A9Aa314B776Ac51A25021C44CAF86bEd', '0xA9F8c770661BeE8DF2D026edB1Cb6FF763C780FF', '0x1Aa53BC4Beb82aDf7f5EDEE9e3bBF3434aD59F12', '0x814CC6B8fd2555845541FB843f37418b05977d8d', '0xc4181dC7BB31453C4A48689ce0CBe975e495321c', '0xC0660932C5dCaD4A1409b7975d147203B1e9A2B6', '0x6E17a8b5D33e6DBdB9fC61d758BF554b6AD93322', '0x295234B7E370a5Db2D2447aCA83bc7448f151161', '0xd0ee89d82183D7Ddaef14C6b4fC0AA742F426355']
   const total_deposited = await api.multiCall({  abi: 'uint256:getTotalDeposited', calls: vaults, permitFailure: true})
   // const total_withdraw = await api.multiCall({  abi: 'uint256:totalAmountToWithdraw', calls: vaults, permitFailure: true})
   const strategies = await api.multiCall({  abi: 'address:strategy', calls: vaults})
   const tokens = await api.multiCall({  abi: 'address:underlyingToken', calls: strategies})
   // const result = total_deposited.map((deposited, index) => deposited - total_withdraw[index]);
+  const staked = await api.call({  abi: 'uint256:totalAssets', target: '0xf073bAC22DAb7FaF4a3Dd6c6189a70D54110525C'})
```

Comment:
can you remove this?
I had a discussion with the team, think it is good to preserve the tvl history of both, if genesis tvl drops to 0 in the future, it is not an issue, we still show uniswap v1 tvl.

Reply:
But the thing is we are using genesis contracts on inception you see. The contracts were rebranded from genesis to inception.

Reply:
yeah, they do show up in the parent together

Reply:
@g1nt0ki I do know that sir. But genesis should be removed since there's no genesis anymore and old genesis contracts are now inception contracts. For example if someone search for the genETH (genesis old token) the contract will return inETH (wich is inception new token) so it doesnt make sense to show this TVL as genesis TVL since it's in fact inception TVL.

Reply:
With the parent we are showing that some TVL is from genesis, when in fact is not genesis TVL but yes InceptionLRT TVL

Reply:
I have updated adapters so, after cutoff date, genesis tvl becomes 0 and is counted as part of inception

Reply:
Hum but will genesis still be showing? We would prefer if it's only inception since genesis no longer exists

Reply:
we can add a hallmark that notes the merger but we like to keep historical data

Reply:
And at least some kind of information about genesis no longer existing. Because rn if people click on genesis profile they can go to website and socials of a project that no longer exists

Reply:
I've updated the url and twitter link of Genesis LRT so that they lead to InceptionLRT, when was the merger made official? I can add a hallmark for this event

Reply:
6th May
Announcement -> https://x.com/Genesis_LRT/status/1787498799335665992

Reply:
@realdealshaman Just checked and this is not live yet! When will it be?

Reply:
It's live now :pray:

Reply:
can we close out this pr?

Reply:
Just fixing the tvl calculation now sir
waiting on @g1nt0ki response for that